molparse/system.py

```diff
@@ -69,6 +70,32 @@ def autoname_chains(self, verbosity=1):
                     f"Ambiguous naming! Multiple chains named {chain.name}!"
                 )
 
+    def ssbond_guesser(self, distance=2.15):
+        """Generate SS bond between SG of CYS according to distances"""
+        import numpy as np
+
+        cysteines=[res for res in self.residues if res.name == "CYS"]
+        start=0
+        end=len(cysteines)
+        for i in range(start, end-1):
+            # Look for x, y, z coordinates of SG for cysteine 1
+            for atom in cysteines[i].atoms:
+                if atom.name == "SG":
+                    sg1_pos = np.asarray(atom.position)
```

Comment:
This loop will go away after you use `get_atom` but even then I think it's important to notice that you don't break the loop after finding the correct atom, which wasted some tiny resources haha